### DISCUSSION TITLE
i have permissions problem in my directus

### DISCUSSION THREAD

**Discussion by Aminhadjkacem** [directus/directus#19269] [Feature Requests]
### Summary

i connected directus with filled postgress data base called Test with datauser called odoo and he's the owner of data base but there's a few tables shows me this error 

### Basic Example

{
  "errors": [
    {
      "message": "You don't have permission to access this.",
      "extensions": {
        "code": "FORBIDDEN"
      }
    }
  ]
}

### Motivation

i need to make a headless cms with directus and i must connect directus to an existant psql database from odoo

### Detailed Design

![image](https://github.com/directus/directus/assets/134654858/1f991908-2055-4b4e-8363-6fe2fccc1065)

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

i don't know 

### Alternatives

client project 

### Adoption Strategy

i don't know

**Comment by rijkvanzanten** [comment]
Closing this as it's not a feature request. For questions, [Discord](https://directus.chat) is the better place 👍🏻
